src/js/smtpserver.h

```diff
@@ -3,6 +3,25 @@
 
 #include "js.h"
 
+#define	TRUE	1
```

Comment:
Plain C does not define a boolean type. I would stick to that and just use either 0 or 1 when I need a "boolean" literal.